packages/plugin-preset/src/browser.tsx

```diff
@@ -2,15 +2,16 @@ import type { BrowserPlugin, Suggestion } from '@package-inspector/core';
 import { Card, Grid, Text } from '@nextui-org/react';
 
 import React from 'react';
-
+// FIXME: should expose this nicer
+import { CardView } from '@package-inspector/ui/components/CardView';
```

Comment:
I wonder if we even need to do this, maybe we only always render suggestions a standard way based off of the name/message/etc rather than letting plugins define a custom car view. 

At least for the landing page "card view" i think this ^ makes sense.

Reply:
Yeah good point, so cardView would return options instead of a react view itself? 

```ts
{
  get cardView(): { title: string, bodyMessage: string, footerMessage: string, id: string }
}
```